app/packages/core/src/components/Grid/Grid.module.css

```
@@ -94,3 +94,105 @@
+.swimlaneSiblings {
+  flex: 1 1 auto;
+  position: relative;
+  min-width: 0;
+}
+
+/* Loading shimmer — single overlay sitting behind cover + siblings,
+ * with a sheen that sweeps strictly left → right via transform
+ * (background-position math is finicky once a mask is in play). The
+ * mask fades the sheen near the cover so the area next to the image
+ * stays visually quiet. */
+@keyframes swimlaneShimmerSheen {
```

Stylelint (17.12.0)

[error] Expected keyframe name "swimlaneShimmerSheen" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)